docs/resource-policy/installation.md

```diff
@@ -103,6 +103,26 @@ along with the default values, for the Topology-aware and Balloons plugins Helm
 | `config`           | <pre><code>ReservedResources:</code><br><code>  cpu: 750m</code></pre>                                                        | plugin configuration data                            |
 
+#### Memtierd
+
+| Name               | Default                                                                                                                       | Description                                          |
+| ------------------ | ----------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------- |
+| `image.name`       | [ghcr.io/containers/nri-plugins/nri-resource-policy-memtierd](ghcr.io/containers/nri-plugins/nri-resource-policy-balloons)    | container image name                                 |
+| `image.tag`        | unstable                                                                                                                      | container image tag                                  |
+| `image.pullPolicy` | Always                                                                                                                        | image pull policy                                    |
+| `resources.cpu`    | 250m                                                                                                                          | cpu resources for the Pod                            |
+| `resources.memory` | 100Mi                                                                                                                         | memory qouta for the                                 |
+
+#### Memory-qos
+
+| Name               | Default                                                                                                                       | Description                                          |
+| ------------------ | ----------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------- |
+| `image.name`       | [ghcr.io/containers/nri-plugins/nri-resource-policy-memory-qos](ghcr.io/containers/nri-plugins/nri-resource-policy-balloons)  | container image name                                 |
+| `image.tag`        | unstable                                                                                                                      | container image tag                                  |
+| `image.pullPolicy` | Always                                                                                                                        | image pull policy                                    |
+| `resources.cpu`    | 10m                                                                                                                          | cpu resources for the Pod                            |
```

Comment:
would it be enough to have it as besteffort container?

Reply:
Probably @askervin can give better answer :)

Reply:
`nri-memory-qos` could indeed be a besteffort container, because it is very-very lightweight task that it is handling.

`nri-memtierd` is another story. Setting resources.limits works infact as a convenient way to make sure that memtierd processes would not consume too much CPU from the node. On the other hand, deciding what would be a proper portion of CPU that should be sufficient, depends heavily on user needs and other workloads (most essentially how much there is memory to scan). Today I would vote for _limiting_ CPU usage to something like 250m or 500m, but not necessarily _requesting_ any CPU, so that deploying the plugin would not be prevented due to exhausted reserved CPU resources on a node. @kad , what do you think?

Reply:
my point was for "memtier-qos" container, which is just very simple, modifies only cgroupsv2 parameters. and thus best effort: neither cpu nor memory request.

for full memtierd, yes some minimal requirements good to have plus some way to limit burstability.

BTW, remove references to "balloons" in the URLs in markdown.